akka-javasdk/src/main/scala/akka/javasdk/impl/SdkRunner.scala

```diff
@@ -546,6 +558,7 @@ private final class Sdk(
   private val viewDescriptors: Seq[SpiViewDescriptor] =
```

Comment:
why isn't viewDescriptors covered by above foreach in the same way as other components?
as it is now, views will be logged as "Unknown component"?

Reply:
fixed